hyperon_das/client.py

```diff
@@ -61,13 +61,15 @@ def _send_request(self, payload) -> Any:
             )
         except exceptions.HTTPError as e:
             with contextlib.suppress(pickle.UnpicklingError):
-                return deserialize(response.content).get('error')
-            das_error(
-                HTTPError(
-                    message=f"HTTP error for URL: '{self.url}' with payload: '{payload}'",
-                    details=str(e),
+                message = deserialize(response.content).get('error')
+
+                das_error(
+                    HTTPError(
+                        message="Please, check if your request payload is correctly formatted.",
```

Comment:
This message is still awful. This is a message to an end user. It shouldn't have terms like `payload` which are totally meaningless to him/her. Put yourself in the skin of a user trying to figure out how to make a query to a remote DAS. What error message would YOU like to see if you provided an ill-formed query? 

Also, I noticed toy override `message` returned by deserialization in line 64. Why doing it? Shouldn't it be appended in the final error message the end user will see?

Reply:
@levisingularity Regarding the error message, I think you can consider creating a more specific message inside query() in the FunctionsClient class, on line 128.

@andre-senna I don' understand your point 

**Also, I noticed toy override message returned by deserialization in line 64...**

I understand that he placed the message returned by the server in the exception returned to the user, as well as details. I mean, he's appending

Reply:
@marcocapozzoli  is correct regarding my comment "Also, I noticed toy override message returned by deserialization ..." I didn't saw line 69. 

But this doesn't change my opinion regarding my first point, OK?

Reply:
Now, I'm raising this exception with an HTTP status code. It's caught within the query function, where I verify the status code and present a better message in the function's context for the user.